hw5/bank.js

```diff
@@ -0,0 +1,94 @@
+let balance = 1000;
+
+alert(' Welcome to the Bank of An, Press W for Withdrawls, D for Deposits, B to see current Balance, or Q to Quit.')
+
+document.onkeydown = function (e) {
+    switch (e.key) {
+            case 'w':
+            withdrawAmt = prompt('how much would you like to withdraw')
+            withdrawAmt = Number(withdrawAmt)
+            if (withdrawAmt < 0) {
+              alert('You must enter a positive number')
+            } else if (withdrawAmt > balance){
+              alert('You cannot withdraw more than your current balance')
+            } else if ((balance - withdrawAmt) <300){
+              if(confirm('Doing this transaction will result in a balance less than $300, are you sure you want to proceed?')){
+                balance = balance - withdrawAmt
+              } else{
+                alert('Withdraw aborted')
+              } } else {
+                balance = balance - withdrawAmt
+              }
+            break;
+        case 'W':
+            withdrawAmt = prompt('how much would you like to withdraw')
+            withdrawAmt = Number(withdrawAmt)
+            if (withdrawAmt < 0) {
+              alert('You must enter a positive number')
+            } else if (withdrawAmt > balance){
+              alert('You cannot withdraw more than your current balance')
+            } else if ((balance - withdrawAmt) <300){
+              if(confirm('Doing this transaction will result in a balance less than $300, are you sure you want to proceed?')){
+                balance = balance - withdrawAmt
+              } else{
+                alert('Withdraw aborted')
+              } } else {
+                balance = balance - withdrawAmt
+              }
+            break;
+        case 'd':
+            depositAmt = prompt('how much would you like to deposit')
+            depositAmt = Number(depositAmt)
+            if (depositAmt>50000){
+              alert('Sorry, we do not support deposits over $50,000')
+            } else if ( depositAmt < 0){
+              alert('Sorry, you cannot deposit negative amounts')
+            } else{
+              balance = balance + depositAmt
+            }
+            break;
+        case 'D':
```

Comment:
I'd use toLowerCase for variables you get from the user, that way you only have to account for one case of the letter in the switch statements.